packages/components/src/Avatar/src/AnonymousAvatar.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnonymousRichIcon } from "@hopper-ui/icons";
+import { type ResponsiveProp, useStyledSystem, type StyledSystemProps, useResponsiveValue } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { type CSSProperties, forwardRef, type ForwardedRef } from "react";
+import { useContextProps } from "react-aria-components";
+
+import type { BaseComponentProps } from "../../utils/index.ts";
+
+import type { AvatarSize } from "./Avatar.tsx";
+import { AvatarContext } from "./AvatarContext.ts";
+import { RichIconAvatarImage } from "./RichIconAvatarImage.tsx";
+
+export const GlobalAnonymousAvatarCssSelector = "hop-AnonymousAvatar";
+
+export interface AnonymousAvatarProps extends StyledSystemProps, BaseComponentProps {
+    /**
+     * Whether or not the avatar is disabled.
+     */
+    isDisabled?: boolean;
+    /**
+     * The size of the avatar.
+     * * @default "md"
+     */
+    size?: ResponsiveProp<AvatarSize>;
+}
+
+function AnonymousAvatar(props: AnonymousAvatarProps, ref: ForwardedRef<HTMLDivElement>) {
+    [props, ref] = useContextProps(props, ref, AvatarContext);
+    const { stylingProps, ...ownProps } = useStyledSystem(props);
+    const {
+        className,
+        size: sizeValue,
+        style,
+        ...otherProps
+    } = ownProps;
+
+    const size = useResponsiveValue(sizeValue) ?? "md";
+
+    const classNames = clsx(
+        className,
+        GlobalAnonymousAvatarCssSelector,
+        "hop-AnonymousAvatar",
```

Comment:
`hop-AnonymousAvatar` is a duplication of GlobalAnonymousAvatarCssSelector and is not needed, you can remove this line